libosmscout/include/osmscout/navigation/DataAgent.h

```diff
@@ -105,6 +115,8 @@ namespace osmscout {
         if (GetSphericalDistance(requestMessage->bbox.GetMinCoord(),
                                  requestMessage->bbox.GetMaxCoord()) > Kilometers(2)){
```

Comment:
I do not like a hardcoded value like this in generic code. The value should be configurable (with a sensible default).